lib/ledger.js

```diff
@@ -47,6 +45,7 @@ bedrock.events.on('bedrock.started', async () => {
     try {
       ({ledgerAgent} = await setupVeresOneLedger());
     } catch(e) {
+      console.log('ERR', e);
```

Comment:
this is a double log. the existing logger should be fine. you might need to run the tests with `--log-level debug` to see the output.

Reply:
Woops, was a debugging statement. Thanks, fixed in ec303d42f6b2fe3e03ddd27793d9f186372bfabf.